src/components/dialogs/delete-dialog.js

```diff
@@ -210,14 +210,17 @@ const DeleteDialog = ({
                 {error !== '' && <Alert severity="error">{error}</Alert>}
             </DialogContent>
             <DialogActions>
-                <Button
+                <CancelButton
                     onClick={handleClose}
                     variant="outlined"
                     disabled={loadingState}
+                    withCustomColor={true}
+                />
+                <Button
+                    onClick={handleClick}
+                    disabled={loadingState}
+                    color={'customButton'}
                 >
-                    <FormattedMessage id="cancel" />
-                </Button>
-                <Button onClick={handleClick} disabled={loadingState}>
                     {(loadingState && <CircularProgress size={24} />) || (
                         <FormattedMessage id="delete" />
```

Comment:
In the US they don't specify to change the delete button. If so, we should do it everywhere no ?

Reply:
After discussion with the PO, we are implementing it here and this component is generic and used wherever there is a deletion